### PR TITLE
Link to french translation

### PR DESCRIPTION
**Pull request by DirtyF**
Work in progress, but advanced enough to get started or upgrade to  version 1.0

I removed all non documentation files except the LICENSE file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -231,8 +231,9 @@ All autotrack plugins are [tested via Sauce Labs](https://saucelabs.com/u/autotr
 
 The following translations have been graciously provided by the community. Please note that these translations are unofficial and may be inaccurate or out of date:
 
-* [Japanese](https://github.com/nebosuker/autotrack)
 * [Chinese](https://github.com/stevezhuang/autotrack/blob/master/README.zh.md)
+* [French](https://github.com/DirtyF/autotrack/tree/french-translation)
+* [Japanese](https://github.com/nebosuker/autotrack)
 * [Polish](https://github.com/krisu7/autotrack)
```
Comment (DirtyF) [inline README.md]: Reordered languages by alphabetical order.

**Comment by philipwalton** [comment]
Did you mean to close this?

**Comment by DirtyF** [comment]
@philipwalton not ar all 😲

**Comment by philipwalton** [comment]
Thanks!

**Pull request merged**